src/button.js

```diff
@@ -158,12 +158,24 @@ export default class Button {
 			ok = options.click.call(this, event);
 		}
 		if (ok) {
+			if (this.widget.getAttribute('data-url') !== this.options.url) {
```

Comment:
What is that three lines? `this.options` already contains all data-attributes and it’s definitely out of the scope of this PR.